### ISSUE TITLE
Package install instructions. 

### ISSUE THREAD

**Issue by seabbs** [publichealthengland/coronavirus-dashboard-api-R-sdk#2]
README indicates that the package is on CRAN but I think this is not actually the case yet? I assume this is in the pipeline but it might be a good idea to amend to the README until this is the case.

**Comment by xenatisch** [comment]
Hi @seabbs 

We submitted the package to CRAN nearly 2 weeks ago - several days before the release. 

They have since rejected it several times for reasons entirely unrelated to the code or the performance of the package - e.g. the description was missing a full stop at the end, or the API link wasn't working (because it was a reference and didn't have any metrics). 

We have addressed them all, even changed API infrastructure to redirect the home page w/o metrics to the developers guide... but we are still waiting. 

I haven't modified the README file to ensure that the package we have submitted to CRAN remains consistent with the one that's on GITHUB... 

Apologies for any trouble. I will contact CRAN again to see what's going on.
